src/PhpSessionPersistence.php

```diff
@@ -125,6 +134,15 @@ public function isNonLocking() : bool
         return $this->nonLocking;
     }
 
+    /**
+     * @internal
```

Comment:
Private methods do not need to be marked `@internal`; they're implicitly internal, as they cannot be called by any but the defining class.

(I'm currently rebasing your patch against develop, and will make this change before pushing.)